crosscomponents/cmakecmd.bat

```diff
@@ -0,0 +1 @@
+cmake -DCMAKE_USER_MAKE_RULES_OVERRIDE=F:\coreclrGit\coreclr\crosscomponents\windows-compiler-override.txt -DCLR_CMAKE_HOST_ARCH=x64 -DCLR_CMAKE_TARGET_ARCH=arm64 -G "Visual Studio 14 2015 Win64"  %1
```

Comment:
It seems this file should not be part of the change (guessing from the hardcoded path)